uast/ann/ann.go

```diff
@@ -53,14 +53,20 @@ type Rule struct {
 	rules      []*Rule
 }
 
+const (
```

Comment:
I wouldn't make this code public, since it's only needed in the `String()` method. I guess you're making it public to reuse it in the tests, which I think it's a bad policy, but feel free to ignore this comment if the opposite is agreed on.